jupyterhub/spawner.py

```diff
@@ -107,9 +148,14 @@ def options_from_form(self, form_data):
         prior to `Spawner.start`.
         """
         return form_data
-    
-    user_options = Dict(help="This is where form-specified options ultimately end up.")
-    
+
+    user_options = Dict(
+        help="""
+        Dict of user specified options specific to this spawned instance of the single-user server.
```

Comment:
Dict of user specified options for the user's spawned instance of a single-user server. These user options are usually provided by an `options_form` displayed to the user for setting options.